cpp/inc/bond/core/schema.h

```diff
@@ -202,7 +213,7 @@ class InitSchemaDef
 
         field.id = id;
         field.metadata = metadata;
-        field.type = GetTypeDef<typename remove_maybe<T>::type>();
+        field.type = GetTypeDef<typename remove_bonded_value<T>::type>();
```

Comment:
It looks to me like the `remove_bonded_value` here means that `type.bonded_type` won't be set properly anymore. Am I missing something?

Reply:
It should work properly I believe (tests are covering these and I looked manually as well). Note, that unlike `remove_bonded`, the new `remove_bonded_value` removes the `bonded` only once (in case of a `bonded` field we receive `bonded<bonded<T>>`). That is the reason why I added this combined trait, instead of adding another `remove_value` and using it in combination with existing `remove_bonded` .